be/src/util/metrics.h

```diff
@@ -414,6 +414,7 @@ class MetricRegistry {
     std::string to_prometheus(bool with_tablet_metrics = false) const;
     std::string to_json(bool with_tablet_metrics = false) const;
     std::string to_core_string() const;
+    std::string to_json_by_name(const std::string& name, const std::string& group) const;
```

Comment:
There is no problem at the code level, but what is the main phenomenon after adding this function?